backend/infrastructure/api/endpoints/product.py

```diff
@@ -0,0 +1,69 @@
+from fastapi import APIRouter, UploadFile, File, HTTPException, Form
+from infrastructure.mongo.product_handler import upload_product, get_product, get_image
+from fastapi.responses import StreamingResponse
+from io import BytesIO
+
+router = APIRouter()
+
+@router.post("/upload")
+async def upload_product_endpoint(
+    name: str = Form(...),
+    price: float = Form(...),
+    country_of_origin: str = Form(...),
```

Comment:
please introduce a model for this, avoid specyfing that much amount of arguments